frontend/tests/playwright/product/pro_user.spec.ts

```diff
@@ -21,13 +18,9 @@ test.describe('Pro user test', () => {
       await page.waitForSelector('.fa-rectangle-pro');
 
       // Assert price on page is lower than step 1
-      const proPriceString = await page
-         .getByTestId('current-price')
-         .first()
-         .innerText();
+      const proPrice = await productPage.getCurrentPrice();
 
-      expect(parseFloat(originalPriceString.replace('$', ''))).toBeGreaterThan(
-         parseFloat(proPriceString.replace('$', ''))
-      );
+      expect(proPrice).not.toEqual(originalPrice);
+      expect(parseFloat(proPrice)).toBeLessThan(parseFloat(originalPrice));
```

Comment:
Why do we need to `parseFloat` again? Isn't the value returned from `getCurrentPrice` already a float?

Reply:
`toFixed` returns a string so if we are trying to compare the numerical values between the prices then we need to convert it back.

I originally did this because I was influenced by this fix [`74674f6` (#1235)](https://github.com/iFixit/react-commerce/pull/1235/commits/74674f6e889da10c76e470d37d57cb793c45a45d), but seeing as it is really only used in one place. I think returning a `number` and having the Devs handle the `precision` would be better.

Reply:
Ah gotcha! I'm good either way 🤷 

Maybe `parseFloat(parseFloat(val).toFixed(2))`?